Source/CombatExtended/CombatExtended/ArmorUtilityCE.cs

```diff
@@ -502,15 +502,10 @@ public static void ApplyParryDamage(DamageInfo dinfo, Thing parryThing)
                 }
 
                 var penAmount = dinfo.ArmorPenetrationInt; //GetPenetrationValue(dinfo);
-                if (TryPenetrateArmor(dinfo.Def, parryThingArmor, ref penAmount, ref dmgAmount, parryThing))
-                {
-                    // Partially penetrating sharp attacks.
-                    if (dinfo.Def.armorCategory == DamageArmorCategoryDefOf.Sharp && dinfo.Amount > dmgAmount)
-                        ApplyParryDamage(GetDeflectDamageInfo(dinfo, dinfo.HitPart, ref dmgAmount, ref penAmount, true), parryThing);
-                }
+                TryPenetrateArmor(dinfo.Def, parryThingArmor, ref penAmount, ref dmgAmount, parryThing);
                 // Deflected attacks
-                else if (dinfo.Def.armorCategory == DamageArmorCategoryDefOf.Sharp)
-                    ApplyParryDamage(GetDeflectDamageInfo(dinfo, dinfo.HitPart, ref dmgAmount, ref penAmount), parryThing);
+                if (dinfo.Def.armorCategory == DamageArmorCategoryDefOf.Sharp)

```

Comment:
This is not quite an equivalent change.  Previously, if ` TryPenetrateArmor...` returned `true`, `ApplyParryDamage` was called with `partialPen` set to `true` (it defaults to `false`).  This version always calls it with `partialPen` set to `true`.  

To preserve the previous behavior, use `bool partialPen = TryPenetrateArmor(...); ApplyParryDamage(..., ref penAmount, partialPen), parryThing);`